test/integration/fixtures/blocks/core__query.serialized.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:query -->
+<!-- wp:query {"query":{"perPage":null,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"taxQuery":null,"parents":[]},"displayLayout":{"type":"list"}} -->
```

Comment:
I couldn't quite work out why this change to serialized markup for the block with empty attributes is required. Is it because of the deprecation updating the attributes? In practice I don't think it really matters that this serializes the default values of the empty state of the block, but was just curious as to why that's happening now 🤔

Reply:
Oooh that's weird, not sure why it's happening 🤔

Reply:
I simplified the fixture a bit and removed some values I'd incorrectly copied from the previous deprecation, but no changes to this file. I guess it should be fine? 🤷

Reply:
Good idea simplifying the fixtures. For this one, I think it's just that what gets serialized for the block's default state has changed — I don't think it's really an issue though since it's just the block attribute's metadata for when it's in the placeholder state. I'll just CC @ntsekouras for visibility, but I don't think it'd be a blocker.